powersimdata/data_access/data_access.py

```diff
@@ -196,6 +195,18 @@ def makedir(self, relative_path):
         target = os.path.join(self.root, relative_path)
         os.makedirs(target, exist_ok=True)
 
+    def remove(self, target, recursive=False):
+        """Remove target using rm semantics
+
+        :param str target: path to remove
+        :param bool recursive: delete directories recursively
+        """
+        if recursive:
+            target = os.path.join(target, "**")
+        files = glob.glob(target, recursive=recursive)
+        for f in files:
+            os.remove(f)
```

Comment:
Does this fail if when trying to remove a directory? Specifically, when it's used in the `delete` module for deleting the `tmp` folder on line 58? I'm not sure it's an edge case we necessarily want to worry about right now (deleting a scenario in a local setup), but might be good to note.

Reply:
Yeah, looks like it would fail due to the glob matching the `tmp/scenario_n` folder itself. Will fix.

Reply:
Done

Reply:
Let's make extra sure we don't accidentally delete the `tmp` folder itself.

Reply:
For sure - I think we're safe there. We have the confirmation prompt by default, and recursive delete is only called from two places which specify the scenario folder within `tmp`.